config/locales/support_interface/personas.yml

```diff
@@ -0,0 +1,34 @@
+en:
+  personas:
+    permissions:
+      admin: This user has both manage_organisations and manage_users permissions.
```

Comment:
maybe we could format the permission names, or maybe put them in `manage_organisations`?